templates/amazons3_service/src/amazons3_service/main.bal

```diff
@@ -0,0 +1,334 @@
+// Copyright (c) 2019 WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
+//
+// WSO2 Inc. licenses this file to you under the Apache License,
+// Version 2.0 (the "License"); you may not use this file except
+// in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing,
+// software distributed under the License is distributed on an
+// "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+// KIND, either express or implied.  See the License for the
+// specific language governing permissions and limitations
+// under the License.
+
+import ballerina/config;
+import ballerina/http;
+import ballerina/log;
+
+import dilmi/amazons3;
```

Comment:
Why do we have the module name as dilmi?

Reply:
Fixed by https://github.com/wso2/ballerina-integrator/pull/261. Thanks.